Maintenance mode (#1143)
owncloud/docs-ocis: modules/ROOT/pages/maintenance/m-mode/maintenance-mode.adoc

= Maintenance Mode
:toc: right
:description: There are a few things to consider when updating/upgrading Infinite Scale.

== Introduction

> {description} The major points are what is maintenance mode and how to initiate it, respectively what are the user impacts.

Reviewer: ```suggestion
{description} The major points are; what is maintenance mode, how to initiate it, and what are the user impacts.
```

== Backend View

* The intention for bringing an instance into maintenance mode is to do updates/upgrades or reconfiguration.
* An instance in maintenance mode is fully shut down and inaccessible by users but will be up and running again in a relatively short period of time.

> * Maintenance mode should be considered to be active only in off-working or off-peak hours.

Reviewer: ```suggestion
* Maintenance mode should be scheduled only in off-working or off-peak hours.
```

* Any necessary reconfigurations/adaptations and tests have been done upfront, most likely with a staging environment. This guarantees short unavailability.
* {empty}
+

> IMPORTANT: To disable ANY access to the instance, you must MANUALLY configure the webserver to respond for  incoming traffic with `503 Service Unavailable`. This type of response is important because the webUI and apps have procedures to react properly to this response. In addition, a static html page can be created to show the current state for users trying to login. Such a page is not part of Infinite Scale, independent of it and must be provided by the admin.

Reviewer 2: ```suggestion
IMPORTANT: To disable ANY access to the instance, you must MANUALLY configure the webserver to respond for incoming traffic with `503 Service Unavailable`. This type of response is important because the webUI and apps have procedures to react properly to this response. In addition, a static html page can be created to show the current state for users trying to login. Such a page is not part of Infinite Scale, independent of it and must be provided by the admin.
```
